project-config.yml

```
@@ -0,0 +1,30 @@
+  - no_of_weekend_nights
+  - no_of_week_nights
+  - lead_time
+  - no_of_previous_cancellations
+  - no_of_previous_bookings_not_canceled
+  - avg_price_per_room
+  - no_of_special_requests
+
+cat_features:
+  - type_of_meal_plan
+  - required_car_parking_space
+  - room_type_reserved 
```

yamllint

[error] trailing spaces

(trailing-spaces)